template.rb

```diff
@@ -187,6 +188,26 @@ def install_koi
   run("rails katalyst_content:install:migrations")
   run("rails katalyst_navigation:install:migrations")
   run("rails action_text:install")
+  install_active_storage
+end
+
+def install_active_storage
+  run("rails active_storage:install")
+
+  # setup active_storage services
+  copy_file("config/storage.yml")
+  gsub_file("config/storage.yml", "[DEFAULT_BUCKET_ID]", "katalyst-#{@app_name}-staging-assets")
```

Comment:
Needs a strategy for connecting this app to the terraform configured bucket name. Could be manual.